node_modules/gh-statsd/README.md

```diff
@@ -0,0 +1,3 @@
+# The Grasshopper Event Engine statsd module
+
+The statsd module is responsible for interacting with Statsd
```

Comment:
![This is your captain speaking](http://vignette3.wikia.nocookie.net/unanything/images/3/34/Captain_Obvious.jpg/revision/latest?cb=20111219154532)